### DISCUSSION TITLE
Missing a sensor for current day export profit

### DISCUSSION THREAD

**Discussion by M0LTE** [BottlecapDave/HomeAssistant-OctopusEnergy#529] [Q&A]
I have the following sensors; as I understand them to be:

Electricity import cost today
`sensor.octopus_energy_electricity_$account_$import-mpan_current_accumulative_cost`

Electricity import cost yesterday
`sensor.octopus_energy_electricity_$account_$import-mpan_previous_accumulative_cost`

Electricity export "profit" yesterday
`sensor.octopus_energy_electricity_$account_$export-mpan_export_previous_accumulative_cost`

The question:

**I am missing, but would like to see, electricity export "profit" today, i.e.:**
`sensor.octopus_energy_electricity_$account_$export-mpan_current_previous_accumulative_cost`

Is this possible?

Thanks.

**Comment by BottlecapDave** [comment]
To my understanding this isn't available via the API. I don't have export to explore properly. In the app, do you see export data via the mini when looking at the day view (e.g. anywhere that isn't the live use pane)

**Reply by M0LTE** [reply]
Hi Dave

Hmm, only for yesterday and earlier. Seems a bit of a miss from Octopus.

I'm very happy to give you my API key privately so you can have a poke around what the API looks like for my account, if you wish.

**Reply by M0LTE** [reply]
I have a Mini, if that matters/helps.
